ticdc/deploy-ticdc.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Deploy TiCDC
+summary: Learn how to deploy TiCDC and the hardware and software recommendations for deploying and running it.
+---
+
+# Deploy TiCDC
+
+This document describes how to deploy a TiCDC cluster and the hardware and software recommendations for deploying and running it. You can either deploy TiCDC with a new cluster or add the TiCDC components to an existing TiDB cluster. Generally, it is recommended that you deploy it using TiUP. In special situations, you can also deploy it using binary.
```

Comment:
Since we don't specify "special situations", it might cause confusion. We can say it in another way.
```suggestion
This document describes how to deploy a TiCDC cluster and the hardware and software recommendations for deploying and running it. You can either deploy TiCDC along with a new TiDB cluster or add the TiCDC component to an existing TiDB cluster. Generally, it is recommended that you deploy TiCDC using TiUP. In addition, you can also deploy it using binary as needed.
```